tests/dummy/public/test-forms/lang-strings/form.ttl

```diff
@@ -0,0 +1,50 @@
+@prefix form: <http://lblod.data.gift/vocabularies/forms/> .
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix displayTypes: <http://lblod.data.gift/display-types/> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix schema: <http://schema.org/>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>.
+
+##########################################################
+# form
+##########################################################
+ext:mainFg a form:FieldGroup.
+
+ext:form a form:Form, form:TopLevelForm ;
+    form:includes ext:input;
+    form:includes ext:inputNL;
+    form:includes ext:inputEN.
+
+##########################################################
+#  Listing property-group
+##########################################################
+ext:mainPg a form:PropertyGroup;
+    sh:order 1 .
+
+##########################################################
+# Field
+##########################################################
+ext:input a form:Field ;
+    sh:name "Title" ;
+    sh:order 10 ;
+    sh:path ext:title ;
+    form:displayType displayTypes:defaultInput ;
+    sh:group ext:mainPg .
+
+ext:inputNL a form:Field ;
+    sh:name "Title (NL)" ;
+    sh:order 10 ;
+    sh:path ext:title ;
+    form:displayType displayTypes:defaultInput ;
+    form:language "nl";
```

Comment:
I understood URIs are preferred over a string literal, but it seems rdflib returns the language as a string, so we would have to map uris to strings somewhere. I didn't look into it yet since I'm not sure which URIs need to be used anyways.

https://linkeddata.github.io/rdflib.js/doc/classes/Literal.html#language

Reply:
Nah I thought they would be URI's but they are tags with a specific syntax. See BCP47 again: https://www.rfc-editor.org/rfc/rfc5646.html#section-2.1

```
A language tag is composed from a sequence of one or more "subtags",
   each of which refines or narrows the range of language identified by
   the overall tag.  Subtags, in turn, are a sequence of alphanumeric
   characters (letters and digits), distinguished and separated from
   other subtags in a tag by a hyphen ("-", [[Unicode](https://www.rfc-editor.org/rfc/rfc5646.html#ref-Unicode)] U+002D).
```

Reply:
I would just rename the predicate `form:languageFilter`

Reply:
form:language is fine after all